src/ert/callbacks.py

```diff
@@ -128,3 +134,46 @@ async def forward_model_ok(
         run_arg.ensemble_storage.unset_failure(run_arg.iens)
 
     return final_result
+
+
+async def load_run_path_realization(
```

Comment:
Why do we need a new function instead of using `forward_model_ok`? If because of `RunArg`, could we just use the new function everywhere?

Reply:
Yes, but I did not want to mess with `forward_model_ok`